### PR TITLE
Update history with new searches

### PR DESCRIPTION
**Pull request by thalytafabrine**
When doing new searches on search 1.x, the history is not being updated.

[workspace](https://thalyta--storecomponents.myvtex.com)

### CODE REVIEW

**Comment by hiagolcm** [comment]
@chrsmutti I think you have suggested to use the `sae-analytics`, am I right?

File: react/components/Autocomplete/index.tsx [hunk 1]
```diff
@@ -135,8 +135,17 @@ class AutoComplete extends React.Component<
     );
   }
 
+  addTermToHistory() {
+    const path = window.location.href.split("_q=");
```
Comment (hiagolcm) [inline react/components/Autocomplete/index.tsx:139]: I recommend using the `useSearchPage` hook to get the search term. It Is a context with all the info related to the search

Comment (thalytafabrine) [reply react/components/Autocomplete/index.tsx:139]: I tried to use this, but it is always returning `undefined` for the Autocomplete component. I think this only works for children of the search result :/

**Comment by chrsmutti** [comment]
> @chrsmutti I think you have suggested using the `sae-analytics`, am I right?

@hiagolcm It can be done in `sae-analytics` or here, here we would still need to have access to the key used to store the history, so maybe it's better here and not in two different places? 

LGTM!

**Pull request merged**